sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/aio/client.py

```diff
@@ -194,10 +196,21 @@ class CertificateClient(AsyncKeyVaultClientBase):
                 :caption: Delete a certificate
                 :dedent: 8
         """
-        bundle = await self._client.delete_certificate(
-            vault_base_url=self.vault_url, certificate_name=name, error_map=_error_map, **kwargs
+        polling_interval = kwargs.pop("_polling_interval", None)
+        if polling_interval is None:
+            polling_interval = 2
+        deleted_certificate = DeletedCertificate._from_deleted_certificate_bundle(
+            await self._client.delete_certificate(
```

Comment:
Style nitpick: I wouldn't nest these calls.

Reply:
ok, I've taken that out, it does look weird ot have the await there. thanks!